src/main/java/com/example/TicketFinder/Concert.java

```diff
@@ -0,0 +1,19 @@
+package com.example.TicketFinder;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+class Concert {
+    private String nameOfConcert;
+    private LocalDate date;
+    private String placeOfConcert;
```

Comment:
What do you mean by place? If it is a an address, please rename the field to `address`